bucket/multipar-beta.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "1.3.2.4",
+    "description": "A tool for creating and using parchive files to detect damage in data files and repair them if necessary.",
+    "homepage": "https://hp.vector.co.jp/authors/VA021385/",
+    "license": "Freeware",
+    "url": "https://github.com/Yutaka-Sawada/MultiPar/releases/download/v1.3.2.4/MultiPar1324.zip",
+    "hash": "md5:752d59de3a91d41939b4040a07b32b0d",
+    "bin": [
+        "MultiPar.exe",

```

Comment:
Does it take any CLI args?

Reply:
Seems not. It's a GUI app with lots of options and `multipar -h` is invalid.

Reply:
Then we can remove it from bin

Reply:
Done. And I also removed "ParQueue.exe" from bin because it also doesn't take any CLI args and seems can't be directly used by users.